openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/5-infra-chaos/RACV-node-failure-cstor/node-failure-cstor

```diff
@@ -0,0 +1,275 @@
+#!/bin/bash
+set -x
+
+pod() {
+### Drain the node to run infra chaos test scripts
+echo "******* Drain node for maintainence ********"
+
+node_name=$(sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port kubectl get nodes | grep compute | awk 'FNR==1 {print $1}')
```

Comment:
Included the taint command in this task. Konvoy cluster doesn't have any compute permission to the nodes.